tests/test_ideogram.py

```diff
@@ -0,0 +1,311 @@
+import os
+from pytest_dash.utils import (
+    wait_for_element_by_css_selector
+)
+from .test_common_features import (
+    init_demo_app,
+    template_test_component,
+    template_test_component_single_prop,
+    PROP_TYPES,
+    COMPONENT_REACT_BASE
+)
+
+# define app name once
+APP_NAME = os.path.basename(__file__).replace(
+    'test_', '').replace(
+        '.py', '').replace(
+            '_', '-')
+
+# pass/fail strings
+PASS = 'PASSED'
+FAIL = 'FAILED'
+
+# define any custom strings here (e.g., if you use
+# a particular CSS selector a lot, assign it to a
+# variable)
+
+
+# Demo app tests
+
+@init_demo_app(APP_NAME)
+def test_click_app_name_from_gallery(dash_threaded, selenium):
+    """Test that clicking on the given app goes to the expected URL."""
+    assert selenium.current_url.replace('http://localhost:8050', '').strip('/') == \
+        'dash-bio/{}'.format(APP_NAME)
+
+
+# below, write tests for initial conditions; they will most likely
```

Comment:
I would take out this boilerplate, especially since there's no `@init_demo_app` decorator here.